docs/create-packages/Symbol-Packages-snupkg.md

```diff
@@ -128,4 +127,6 @@ The .nupkg file would be exactly the same as it is today, but the .snupkg file w
 
 ## See Also
 
-[NuGet Package Debugging & Symbols Improvements](https://github.com/NuGet/Home/wiki/NuGet-Package-Debugging-&-Symbols-Improvements)
+Consider using Source Link to enable source code debugging of .NET assemblies. For more information, please refer to the [Source Link guidance](/dotnet/standard/library-guidance/sourcelink.md).
```

Comment:
Is the sourcelink URL correctly configured for relative hit?

Reply:
may be use absolute url?

Reply:
Talked offline, this is the pattern used to reference a docs page. We can't test the URL is correct until after we've merged sadly :/